miasm2/analysis/depgraph.py

```diff
@@ -759,15 +759,13 @@ def _follow_mem_wrapper(exprs, mem_read):
             follow.update(expr.get_r(mem_read=mem_read, cst_read=True))
         return follow, set()
 
-    @staticmethod
-    def _follow_mem(exprs):
+    def _follow_mem(self, exprs):
```

Comment:
My remark only concerns `_follow_mem` and `_follow_nomem`, as they both call `_follow_mem_wrapper`, which is a static method in both `depgraph` and `influencegraph`.

But your note still stands for call to `copy` and `extend` in influence graph.